app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -218,6 +218,9 @@ def render_unavailable
         message = I18n.t("curation_concerns.workflow.unauthorized")
         respond_to do |wants|
           wants.html do
+            if @presenter.blank?
```

Comment:
@jcoyne @cam156 How would you feel about:

1. Putting line 222 behind its own private message, one named something like `presenter_for_not_found_concern`; and
2. Invoking the new method within the `document_not_found!` method, which will make sure `@presenter` gets set right next to the code that handles the "not found" case

:question: